Packs/Phishing/ReleaseNotes/3_5_17.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### DeleteReportedEmail
+
+- Fixed an issue where if context data about the original E-Mail is missing, the script would fail with a vague error instead of a proper error message. 
```

Comment:
```suggestion
- Fixed an issue where if context data about the original email is missing, the script would fail with a vague error instead of a proper error message. 
```